app/controllers/profiles/sessions_controller.rb

```diff
@@ -0,0 +1,25 @@
+class Profiles::SessionsController < Devise::SessionsController
+# before_action :configure_sign_in_params, only: [:create]
+
+  # GET /resource/sign_in
+  def new
+    super
```

Comment:
As before, I don't think you need these supers (unless it's just for documentation reasons?)